README.md

```diff
@@ -314,6 +314,7 @@ cd epidose/epidose/device
 sudo ansible-playbook install_and_configure.yml --tags "production|development" --extra-vars "eduroam_network_psk=password_of_the_epidose_eduroam_account"
 cd /home/epidose/epidose/epidose/device
 sudo ansible-playbook install_and_configure.yml --tags "delete"
+sudo reboot
```

Comment:
Just ask the user to do it through instructions.  It's rude to reboot the device, because important output may get lost.

Reply:
But these are instructions for the users.
Or do you want me to just write that a user should reboot the device to apply the changes?

Reply:
Sorry, I misunderstood.